install_files/ansible-base/roles/tor-hidden-services/tasks/configure_tor_hidden_services.yml

```diff
@@ -29,7 +29,7 @@
     owner: "{{ tor_user }}"
     group: "{{ tor_user }}"
     mode: "0700"
-  with_items: "{{ tor_auth_instances_v3 }}"
+  with_items: "{{ tor_instances_v3 }}"
```

Comment:
@kushaldas this change resolves the problem you describe here: https://github.com/freedomofpress/securedrop/pull/4652#issuecomment-521395752